cgsmiles/graph_layout_utils.py

```diff
@@ -276,11 +235,22 @@ def _optimize_geometry_2D(positions,
 
     Parameters
     ----------
-    pos: np.ndarray((n, 2))
+    positions: np.ndarray((n, 2))
         the inital positions
+    atom_to_idx: dict
+        dict mapping node keys to indices in the position
+        array
     interactions: dict[str, list(tuple)]
         a dict of interactions with key type and list of
         interaction tuples
+    inter_methods: dict
+        dict of interaction methods
+    neighbors:
+        (n x n) boolen array where direct neighbors
+        are False
```

Comment:
False? Shouldn't neighbors be True? In addition, you *generate* the neighbor index array, so why also accept it as argument??

Reply:
Yep the docstring is wrong; but the neighbor index array is not generated in this function

Reply:
Oh, missed the code fold marks